src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -20,5 +21,28 @@ public IObservable<IPullRequestModel> CreatePullRequest(IRepositoryHost host, IS
 
             return host.ModelService.CreatePullRequest(repository, title, body, source, target);
         }
+
+        public string GetPullRequestTemplate(ISimpleRepositoryModel repository)
+        {
+            Extensions.Guard.ArgumentNotNull(repository, nameof(repository));
+
+            var paths = new[]
+            {
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE.md"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE.md"),
```

Comment:
It's less about readibility and more about this being a really wasteful way to do it, every time this method gets called a new array gets allocated and strings get allocated and concatenated, when we can just do it once for the names of the files and one Combine when we check, and if we would want to add more filenames to check we would only change the data (the array) and not the logic (the method). Also, this service is 30 lines, it's not going to be that hard to see the filenames if they're outside the method...